Dockerfile-infra

```diff
@@ -33,7 +33,7 @@ RUN set -eux \
 
 RUN set -eux \
 	&& apk add --no-cache postgresql-dev postgresql-libs \
-	&& pip3 install --no-cache-dir --no-compile docker pexpect psycopg2 pypsexec pymongo PyMySQL smbprotocol \
+	&& pip3 install --no-cache-dir --no-compile docker docker-compose pexpect psycopg2 pypsexec pymongo PyMySQL smbprotocol \
```

Comment:
Why is `docker-compose` required inside the container?

Reply:
I've used ansible compose module, so it's require this pip package. I think it would be useful to some people.